themes/edge.edx.org/lms/static/sass/partials/base/_certificates.scss

```diff
@@ -0,0 +1,27 @@
+// Certificate overrides for edge.edx.org
+
+.certificate-card {
+    &.mode-honor, &.mode-audit {
+        border-color: $honor-mode-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/honor.png');
+        }
+    }
+
+    &.mode-verified {
+        border-color: $verified-mode-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/verified.png');
+        }
+    }
+
+    &.mode-professional {
```

Comment:
We have `professionalcert-program.png` and `professional-education.png`.  Are these two things?  Is `mode-professional` too generic, or does it cover both?

Reply:
mode-professional only covers the course certificate. The program certificate icon is not being used yet, but will be added at some point. I will rename professional-education.png as just professional.png.

Reply:
I've also renamed professionalcert-program.png to be professional-program.png.

Reply:
NIT / optional: if it helps, courses are professional-ed, and professional-certificate-program is technically the full name for the program.

Reply:
@andy-armstrong @marcotuts: in the DDD book we are reading and with past experience, I think using accurate and consistent names should not be considered a nit. Is there a term "professional" that is meant to apply to both? If not, I'd loop back and fix this.